README.md

```diff
@@ -15,21 +15,24 @@ Setup for developers (Unix)
 ---------------------------
 
 1. Make sure you have installed Python 3.6, [pip3](https://pip.pypa.io/en/latest/) and [virtualenv](http://www.virtualenv.org/en/latest/).
```

Comment:
python3.7 is also supported now, can you update in reference of this.

Reply:
Updated!